README.md

```diff
@@ -64,3 +71,8 @@ TODO: Channel config.
   "birthDate": "1981-05-21"
 }
 ```
+
+You should receive the created form of the Patient resource in HAPI FHIR - notice the Resource now has an `id` field.
+
+To **update** a patient, send through the payload above to `/patient-registration/{FHIR_ID}` but add the field `id` to the object root.
+Also substitute in the `id` into the Request path in the `FHIR_ID` placeholder position.
```

Comment:
@bradsawadye Could you please add some examples of the Patient search feature here as well as the immunization